spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/core/convert/DatastoreEntityConverterImplTests.java

```diff
@@ -0,0 +1,176 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.datastore.core.convert;
+
+import com.google.cloud.Timestamp;
+import com.google.cloud.datastore.Blob;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.LatLng;
+import com.google.cloud.datastore.testing.LocalDatastoreHelper;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.DatastoreMappingContext;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author Dmitry Solomakha
+ *
+ * @since 1.1
+ */
+
+public class DatastoreEntityConverterImplTests {
```

Comment:
I think you should add tests for those two cases where we should throw exceptions:

- when the read value is null (can't be assigned to the needed type)
- when the value can't be written using the Value binder methods in that map